core/src/test/java/com/yahoo/tagchowder/ParserTest.java

```diff
@@ -51,6 +53,98 @@ public void testSampleHtml() throws IOException, SAXException {
         parser.parse(inSource);
     }
 
+    /**
+     * Parse an sample AMP html.
+     *
+     * @throws IOException IOException
+     * @throws SAXException SAXException
+     */
+    @Test
+    public void testSampleAMPHtml() throws IOException, SAXException {
+        final String html = getSampleHtml("amphtml.txt");
+        final Parser parser = new Parser();
+        final CustomHandler customHandler = new CustomHandler();
+        parser.setContentHandler(customHandler);
+        parser.setErrorHandler(customHandler);
+        parser.setFeature(Parser.AMP_VALIDATION_FEATURE, true);
```

Comment:
have another UT with AMP_VALIDATION_FEATURE=true for html.txt and verify attributes match what you expect (such as html and ParsedHtmlTag)

Reply:
Done